plugin/clone/include/clone_os.h

```diff
@@ -134,4 +134,10 @@ inline uchar *clone_os_align(uchar *pointer) {
   return (aligned_ptr);
 }
 
+inline size_t pad_for_o_direct(size_t buf_len, bool o_direct) {
```

Comment:
Maybe just a quick comment on what this function is trying to do given the input lengths? It seems all of the other functions in this header file have a comment. Should this follow the `clone_os_` prefix convention of the other functions?

Reply:
Done and done